es-ds-components/components/es-error-page.vue

```diff
@@ -4,7 +4,21 @@ import IconCommunitySolar from './icon/community-solar.vue';
 import IconInfoBlogPost from './icon/info-blog-post.vue';
 import IconMarketplace from './icon/marketplace.vue';
 
-const ES_DOMAIN = process.env.ES_DOMAIN || 'https://www.energysage.com';
+const props = defineProps({
+    errorType: {
+        type: String,
+        required: true,
+    },
+    useEsDomain: {
+        type: Boolean,
```

Comment:
What I meant was an optional string, maybe `esDomain` that would be used directly, and then we don't use `process.env.ES_DOMAIN` at all, we use `props.esDomain` directly, or the energysage.com string as a backup.

Reply:
I changed it to a optional string prop.

I can see the appropriate URL in the links when I test the new parameter on the docs page http://localhost:8500/organisms/error-page